src/flashbar/interfaces.ts

```diff
@@ -35,6 +36,7 @@ export interface FlashbarProps extends BaseComponentProps {
    * * `dismissible` (boolean) - Determines whether the component includes a close button icon. By default, the close button is not included.
    * When a user clicks on this button the `onDismiss` handler is called.
    * * `dismissLabel` (string) - Specifies an `aria-label` for to the dismiss icon button for improved accessibility.
+   * * `statusIconLabel` (string) - Specifies an `aria-label` for to the status icon for improved accessibility.
```

Comment:
"Specifies an 'aria-label' for the status icon, to improve accessibility."

Reply:
Should we opt for consistency here? The current phrasing matches the text from `dismissLabel` above.